SimPEG/regularization/base.py

```diff
@@ -130,11 +296,19 @@ def __call__(self, m):
 
             r(m) = \\frac{1}{2}
         """
-        r = self.W * (self.mapping * (self._delta_m(m)))
+        r = self.W * self.f_m(m)
```

Comment:
should we use this as an opportunity to remove the 1/2 here and in the data misfit to address #763? I don't think we should do it in this pr, but it seems like a complementary change to make in either the same release or a new release very quickly after this. To keep conversations scoped, we can continue discussion on #763 and keep this pr focussed on the reg, but reading the doc strings prompted this

Reply:
I am easy either way. Guess once you review the math with your group, you can confirm if it still adds up when those halves are removed, then we just pull them once and for all?